components/PublishUpdateBtnWithData.js

```diff
@@ -19,15 +22,86 @@ class PublishUpdateBtn extends React.Component {
   constructor(props) {
     super(props);
     this.onClick = this.onClick.bind(this);
+    this.state = {
+      updateNotificationAudience: 'Financial Contributors',
+    };
   }
 
   async onClick() {
     const { id } = this.props;
-    await this.props.publishUpdate(id);
+    const { updateNotificationAudience } = this.state;
+    await this.props.publishUpdate(id, updateNotificationAudience);
+  }
+
+  handleNotificationChange(selected) {
+    this.setState({ updateNotificationAudience: `${selected.value}` });
   }
 
   render() {
     const update = this.props.data.Update;
+    const isHost = get(update, 'collective.isHost');
+    const backers = get(update, 'collective.stats.backers.all');
+    const hostedCollectives = get(update, 'collective.stats.collectives.hosted');
+
+    const options = [
+      {
+        label: (
+          <FormattedMessage id="update.notify.financialContributors" defaultMessage="Notify financial contributors" />
+        ),
+        value: 'Financial Contributors',
```

Comment:
As per the API feedback, the `value` fields in this list should match the enum values (ie. `FINANCIAL_CONTRIBUTORS`)